tracingganttchart/src/TracingGanttChart/utils.ts

```diff
@@ -68,3 +69,15 @@ export function formatDuration(timeMs: number): string {
   }
   return `${+(timeMs / 1000).toFixed(2)}s`;
 }
+
+export function renderTemplate(
```

Comment:
do we already have this utility in the core or plugin system packages? it seems something we probably use already for queries